### DISCUSSION TITLE
ppt viewer

### DISCUSSION THREAD

**Discussion by Devin7734** [community/community#120847] [Programming Help]
### Select Topic Area

Question

### Body

i want in react js ppt viewer pages

**Comment by LiteBrite82** [comment]
Hi @Devin7734 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project [facebook/react](https://github.com/facebook/react). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by gurjeetsinghvirdee** [comment]
Hey @Devin7734 

Go through this blog, it will help you https://pspdfkit.com/blog/2023/how-to-build-a-react-powerpoint-viewer/

**Reply by ebndev** [reply]
Hey @Devin7734! Were you able to check out this article and try to build it?
